stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -1007,3 +1007,58 @@ function matmul3x3!(C::AbstractMatrix, tA, tB, A::AbstractMatrix, B::AbstractMat
     end # inbounds
     C
 end
+
+# Three-argument *
+*(A::AbstractMatrix, B::AbstractMatrix, x::AbstractVector) = _mat_mat_vec(A,B,x)
+
+*(α::Number, A::AbstractMatrix, x::AbstractVector) = _scalar_mat_vec(α,A,x)
+*(α::Number, A::AbstractMatrix, B::AbstractMatrix) = _scalar_mat_mat(α,A,B)
+*(A::AbstractMatrix, x::AbstractVector, α::Number) = _scalar_mat_vec(α,A,x)
+*(A::AbstractMatrix, B::AbstractMatrix, α::Number) = _scalar_mat_mat(α,A,B)
+
+*(α::Number, A::AbstractArray, β::Number) = (α*β) * A
```

Comment:
For non-commutative numbers, this will be wrong in general. I'd suggest to use
```suggestion
*(α::Number, A::AbstractArray, β::Number) = broadcast(*, α, A, β)
```
as you have below.

Reply:
Will do, this one is easy.